src/Mailgun/Model/MailingList/MailingList.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\MailingList;
+
+/**
+ * @author Michael Münch <helmchen@sounds-like.me>
+ */
+final class MailingList
+{
+    /**
+     * @var string
+     */
+    protected $name;
```

Comment:
There is no need to set these properties as `protected`, this class is flagged as `final` so all of them can be converted to `private`.

We have this case on other models too. If you can update them at the same time please would be great! Thanks,